reference/auth/models/coastalresponse.v1.yaml

```diff
@@ -0,0 +1,21 @@
+title: Coastal Response
+description: Data retrieved from Coastal's API.
```

Comment:
This too could be improved with details. Which Coastal API? For what purpose? To what end?